CHANGELOG.md

```diff
@@ -2,8 +2,9 @@
 UNRELEASED
 ===================
  * see https://github.com/clux/kube-rs/compare/0.54.0...master
+ * `kube` BREAKING: when opting out of default-features, you need to specify `client` feature
```

Comment:
```suggestion
```
Not a breaking change anymore (can just add `native-tls` or `rustls-tls` like before), but it might be a good idea to mention the internal change. And also mention that "core" version can be used by disabling default features.

One breaking change: Now errors if `PatchParams::force` is used when not using `Patch::Apply` instead of logging a warning.

Reply:
ah, good catch. updated. I think the last is possibly more of a bug fix, but left a note on it.